pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -190,6 +186,7 @@ qtModule {
 
   buildInputs = optionals stdenv.isDarwin (with darwin; [
```

Comment:
darwin shouldn't be used as an input.

Reply:
Noted, but it's a big refactoring, better come in a separate PR.